script/entrypoint.sh

```diff
@@ -10,6 +10,7 @@ pip install -r modules-requirements.txt
 #pip install -e /openimis-be-calculation_comores_py
 #pip install -e /openimis-be-report-comores_py
 #pip install -e /openimis-be-core_py
+pip install -e /openimis-be-location_py
```

Comment:
Attention , ligne inutile.
Il faut mettre en commentaire, mais pour appliquer cette install dans le build il faudra mettre a jour la version du package. C'etait normalement la 1.7.0 , il faudra preciser dans le module be location que ca sera la 1.7.1 pour qu'il install automatiquement la nouvelle version